cmd/tdexd/main.go

```diff
@@ -63,7 +66,16 @@ func main() {
 	marketRepository := dbbadger.NewMarketRepositoryImpl(dbManager)
 	tradeRepository := dbbadger.NewTradeRepositoryImpl(dbManager)
 
-	explorerSvc := explorer.NewService(config.GetString(config.ExplorerEndpointKey))
+	var explorerSvc explorer.Service
+	if rpcEndpoint := config.GetString(config.ElementsRPCEndpointKey); len(rpcEndpoint) > 0 {
```

Comment:
I think this should live in the Validate method of the config package

Reply:
do you mean a `config.GetExplorerService()` getter, for instance?

Reply:
Like we do for the SSL check. A specific getter also could be useful

Reply:
the correctness of the Elements endpoint is checked into the `elements.NewService()` factory function... furthermore, if we add a `config.GetExplorer()` getter that takes care of instantiating the correct explorer depending on the env vars, we would still check the correctness of the endpoint at config level, but without repeating the same checks twice.